### PR TITLE
Add mavenCentral() configuration for publish

### PR DESCRIPTION
**Pull request by alorma**
Add configuration to publish to **mavenCentral**, used this blog post: https://getstream.io/blog/publishing-libraries-to-mavencentral-2021/

### CODE REVIEW

**Comment by alorma** [comment]
Blocking it until we have the configuration (gpg key, sonatype account...)

File: scripts/publish-module.gradle [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                        }
+                        developer {
+                            id = 'sloy'
+                            name = 'Rafa Vazquez'
+                            email = 'rafa.vazsan@gmail.com'
+                        }
+                        developer {
+                            id = 'rocboronat'
+                            name = 'Roc Boronat'
+                            email = 'pipobcn@gmail.com'
```
Comment (rocboronat) [inline scripts/publish-module.gradle:67]: May I use my professional email? Or is it bound somewhere with the personal one?

Comment (alorma) [reply scripts/publish-module.gradle]: don't worry, I've deleted this part and used only organization.. hope is not mandatory to have a `developer`

Comment (rocboronat) [reply scripts/publish-module.gradle]: Cool :·)

**Review by Sloy — APPROVED** [review]

**Pull request merged**